modules/govuk/manifests/apps/email_alert_api.pp

```diff
@@ -261,6 +271,7 @@
       value   => $publishing_api_bearer_token;
   }
 
+  # TO BE DELETED:
```

Comment:
It'd be good to say when this can be deleted so you know when/if you can do the deletion.